test/test_framework.py

```diff
@@ -866,24 +885,28 @@ def setUp(self):
         self.addCleanup(shutil.rmtree, str(self.tmpdir))
 
     def test_stored_dict_repr(self):
-        self.assertEqual(repr(ops.StoredDict(None, {})), "ops.framework.StoredDict()")
+        self.assertEqual(repr(ops.StoredDict(None, {})),  # type: ignore
+                         "ops.framework.StoredDict()")
         self.assertEqual(
             repr(
                 ops.StoredDict(
-                    None, {
+                    None, {  # type: ignore
```

Comment:
Not your issue, but the formatting here is weird! Seems like it should be:

```python
self.assertEqual(repr(ops.StoredDict(None, {"a": 1})),  # type: ignore
                 "ops.framework.StoredDict({'a': 1})")
```

Feel free to fix it in a drive-by here or in a separate commit.